config/nightwatch.php

```diff
@@ -6,6 +6,8 @@
     'deployment' => env('NIGHTWATCH_DEPLOY'),
     'server' => env('NIGHTWATCH_SERVER', (string) gethostname()),
     'capture_exception_source_code' => env('NIGHTWATCH_CAPTURE_EXCEPTION_SOURCE_CODE', true),
+    'capture_request_body' => env('NIGHTWATCH_CAPTURE_REQUEST_BODY', false),
```

Comment:
Technically this should probably be `capture_request_body_on_unhandled_exception`, but it felt a bit wordy, and I wondered whether there may be other scenarios we'd capture the body for.